PURL-TYPES.rst

```diff
@@ -280,6 +280,20 @@ hex
       pkg:hex/bar@1.2.3?repository_url=https://myrepo.example.com
 
 
+huggingface
```

Comment:
@McPatate would you be able to do a quick review or loop in someone else from Hugging Face to verify I got the various model identifiers right?